cmd/gateclient/client.go

```diff
@@ -249,7 +217,7 @@ func (m *ApiMeta) InitializeClient() (*http.Client, error) {
 		}
 	} else if auth != nil && auth.Enabled && auth.Basic != nil {
 		if !auth.Basic.IsValid() {
-			return nil, errors.New("Incorrect Basic auth configuration. Must include username and password.")
+			return nil, errors.New("incorrect Basic auth configuration. Must include username and password")
```

Comment:
Is there a particular reason for the punctuation change?

Reply:
not critical
https://github.com/golang/go/wiki/CodeReviewComments#error-strings